portal/static/js/main.js

```diff
@@ -467,7 +467,7 @@ var tnthAjax = {
     },
     "sendRequest": function(url, method, userId, params, callback) {
         if (!url) { return false; }
-        var defaultParams = {attempts: 0, max_attempts: 3, contentType: "application/json; charset=utf-8", dataType: "json", cache: "false", sync: false, timeout: 5000, data: null, useWorker: false, async: true};
+        var defaultParams = {attempts: 0, max_attempts: 3, contentType: "application/json; charset=utf-8", dataType: "json", cache: false, sync: false, timeout: 5000, data: null, useWorker: false, async: true};
```

Comment:
I think `cache: false` might be a little heavy-handed, but out of scope for this review.

The browser should be able to know whether the cache is stale by HTTP headers and if there's stale cache issues we should address them there.